src/style.css

```diff
@@ -1 +1,56 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Don't reset your margins and paddings with * selector, better move it to reset.css and reset them for specific tags